paper.md

```diff
@@ -0,0 +1,160 @@
+---
+title: '[libtrixi]{.sc}: an interface library for using Trixi.jl from C/C++/Fortran'
+tags:
+  - numerical simulation
+  - Julia
+  - cross-language interoperability
+  - computational fluid dynamics (CFD)
+  - high performance computing (HPC)
+authors:
+  - name: Michael Schlottke-Lakemper
+    orcid: 0000-0002-3195-2536
+    corresponding: true
+    affiliation: "1, 2" # (Multiple affiliations must be quoted)
+  - name: Benedict Geihe
+  - orcid: 0000-0002-9519-6872
+    affiliation: 3
+  - name: Gregor J. Gassner
+    orcid: 0000-0002-1752-1158
+    affiliation: 3
+affiliations:
+ - name: Applied and Computational Mathematics, RWTH Aachen University, Germany
+   index: 1
+ - name: High-Performance Computing Center Stuttgart (HLRS), University of Stuttgart, Germany
+   index: 2
+ - name: Department of Mathematics and Computer Science, University of Cologne, Germany
+   index: 3
+date: 01 October 2023
+bibliography: paper.bib
+---
+
+
+# Summary
+
+[`Libtrixi`](https://github.com/trixi-framework/libtrixi) [@schlottkelakemper2023libtrixi]
```

Comment:
Capitalized `Libtrixi` here because it is the beginning of a sentence, right?
Non-capitalized in title (line 2) because that is the way it is really written?
(I'm fine with that!)